public/js/default-text.js

```diff
@@ -73,8 +73,36 @@ var prepText = 'FISH simulates fishing in an ocean. ' +
     'Click on the Go Fishing button on the right when you are ready. ' +
     'Once all the fishers have clicked this button, ' +
     'the first season will begin. ' +
-    '(You may have to wait briefly for all the others fishers to click the button.)';
+    '(You may have to wait briefly for all the other fishers to click the button.)';
 
 var endTimeText = 'Seasons come and seasons go, but for now we are done.';
 
-var endDepletedText = 'All the fish are now gone.';
\ No newline at end of file
+var endDepletedText = 'All the fish are now gone.';
+
+var catchIntentPrompt1 = 'How many fish do you plan to catch in the next season?';
+var catchIntentPrompt2 = '(Optional, and we won\'t hold you to it!)';
+
+// REDIRECTION FEATURE
+
+var explainRedirectText = '<h4>FISH REDIRECTION FEATURES</h4>' +
+    'If this FISH experiment is part of a chain, you will want to sent participants into FISH ' +
+    'without having them log in to FISH in the usual form-based way. ' +
+    'Provide the platform before FISH the standard FISH URL but with 2 query parameters:\n' +
```

Comment:
Nit: I found this slightly confusing. How about `Provide the source platform with the standard FISH URL...`?